src/App.js

```diff
@@ -2,9 +2,9 @@ import './App.css';
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import BookList from './components/BookList';
-import NavBar from './components/navBar';
-import Categories from './components/categories';
-import Footer from './components/footer';
+import NavBar from './components/NavBar';
+import Categories from './components/Categories';
+import Footer from './components/Footer';
```

Comment:
- [ ] As you have changed the names of the modules here, Kindly you may consider also changing the files' names.

![image](https://user-images.githubusercontent.com/97100091/208971208-c733b161-d849-4c2e-bef9-8d7a7ec11c1c.png)